src/windows/models/properties_model.py

```diff
@@ -237,11 +235,6 @@ def remove_keyframe(self, item):
                     log.debug("Found point to delete at X=%s" % point_to_delete["co"]["X"])
                     keyframe["Points"].remove(point_to_delete)
 
-            # Reduce # of clip properties we are saving (performance boost)
-            clip_data = {property_key: clip_data[property_key]}
-            if object_id:
-                clip_data = {'objects': {object_id: clip_data}}
-
```

Comment:
@edumucelli That's definitely not right, `object_id` and the code around it was only added with the OpenCV changes earlier this year (or maybe the very end of 2020). Which is also when this code was broken.

(The _previous_, very-similar code did in fact do the right thing, reducing the `clip.data` `dict`'s contents directly. Whereas this code accidentally _replaces_ `clip_data` (a variable that initially contains a reference to `clip.data`) with the new, reduced `dict`. `clip_data` is then garbage-collected and the optimized `dict` with it, while `clip.data` remains unoptimized. ...But that part of this code is a relatively recent situation.)

Reply:
My mistake, `clip_data` is initially a reference to **`c.data`**, which the old code updated directly when optimizing. Everything else I said holds.

Reply:
I am refraining to change behavior on this PR, wanted only to remove what was unused, but it seems that unused code is actually made my mistake or is untested and broken, so it is difficult to know what is unused and what is unused because it is broken. Should this once again revert the previous behavior?

```
```
if object_id:
  c.data = {'objects': {object_id: clip_data}}
else:
  c.data = clip_data
```